README.md

```diff
@@ -579,7 +581,12 @@ oneview_volume 'Volume_1' do
   storage_pool <storage_pool_name>
   volume_template <volume_template_name>
   snapshot_pool <snapshot_pool_name>
-  action [:create, :create_if_missing, :delete]
+  properties <volume_properties>                # Hash. Used in create_from_snapshot action only. Only available on API500 and onwards.
+  device_volume_name <device_volume_name>       # String. Name of the external volume that want to add. Used in add action only. Only available on API500 and onwards.
```

Comment:
"# String. Name of external volume to be added. Used only in `add` action. Available on API500 and onwards."